### PR TITLE
{bio}[GCC/11.3.0] BLAT v3.7

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Quite some changes:
* sources of newer versions are published in a new URL of the same university
* add many new/missing dependencies
* cleanup the installation directory from sources and build objects, just install the executables
* run tests

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/b/BLAT/BLAT-3.7-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# because it is statically linked and the newer HTSlib in this toolchain is not
+# API compatible with it.
+dependencies = [
+    ('freetype', '2.12.1'),
+    ('libiconv', '1.17'),
+    ('libpng', '1.6.37'),
+    ('MariaDB', '10.9.3'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('zlib', '1.2.12'),
+]
```
Comment (Micket) [inline easybuild/easyconfigs/b/BLAT/BLAT-3.7-GCC-11.3.0.eb:39]: I think this also needs a `('util-linux', '2.38'),` dependency

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/b/BLAT/BLAT-3.7-GCC-11.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/b/BLAT/BLAT-3.7-GCC-11.3.0.eb:40]: indeed, thanks, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17403/commits/4b98d98d7b482aec31c289133bb798f0f7ae234d

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/f0a7622014517c668327f2d44d881bf8 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @lexming!

**Comment by smoors** [comment]
isn't it enough to add `util-linux` as a *builddep* instead of a dep?
i don't see any of the binaries linking to anything that util-linux provides?

**Pull request merged**